ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml

```
@@ -153,6 +153,14 @@ tests:
         pod should start\| pod sysctls\| build volumes should mount given secrets
         and configmaps into the build pod
     workflow: openshift-e2e-ibmcloud-heterogeneous
+- as: ocp-e2e-powervs-ovn-multi-ppc64le
+  cron: 20 11,23 * * *
+  steps:
+    cluster_profile: ibmcloud-multi-ppc64le
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    workflow: openshift-e2e-powervs-upi-cluster
```

```suggestion
    workflow: openshift-e2e-powervs-upi-heterogeneous-cluster
```

Typical pattern for the multiarchitecture compute is to have heterogeneous in the workflow name.